lib/adal-angular.js

```diff
@@ -123,9 +123,9 @@
                                         updateDataFromCache(_adal.config.loginResource);
                                         $rootScope.userInfo = _oauthData;
                                         // redirect to login requested page
-                                        var loginStartPage = _adal._getItem(_adal.CONSTANTS.STORAGE.START_PAGE);
+                                        var loginStartPage = _adal._getItem(_adal.CONSTANTS.STORAGE.LOGIN_REQUEST);
                                         if (loginStartPage) {
-                                            $location.url(loginStartPage);
+                                            $window.location = loginStartPage;
```

Comment:
so we basically want to trigger the locationchange event for the normal full login redirect?